tools/storybook/package.json

```diff
@@ -48,12 +50,71 @@
 		"@storybook/react": "7.6.19",
 		"@storybook/react-webpack5": "7.6.19",
 		"@storybook/theming": "7.6.19",
+		"@woocommerce/admin-library": "workspace:*",
+		"@woocommerce/block-library": "workspace:*",
 		"@woocommerce/eslint-plugin": "workspace:*",
 		"react": "18.3.1",
 		"react-dom": "18.3.1",
 		"storybook": "7.6.19",
 		"typescript": "^5.3.3",
 		"webpack": "^5.89.0",
 		"wireit": "0.14.3"
+	},
+	"wireit": {
+		"build:project:storybook": {
+			"command": "storybook build -c ./.storybook --quiet && pnpm run copy-blocks-storybook-iframe",
+			"env": {
+				"STORYBOOK": "true",
+				"BABEL_ENV": "storybook"
+			},
+			"clean": "if-file-deleted",
+			"files": [
+				".storybook/*.{js,jsx,ts,tsx,html}",
+				"import-wp-css-storybook.sh",
+				"../../packages/js/components/src/**/stories/*.story.@(js|tsx)",
+				"../../packages/js/experimental/src/**/stories/*.story.@(js|tsx)",
+				"../../packages/js/onboarding/src/**/stories/*.story.@(js|tsx)",
+				"../../packages/js/product-editor/src/**/*.(stories|story).@(js|tsx)",
+				"../../plugins/woocommerce-admin/client/**/stories/*.story.@(js|tsx)"
+			],
+			"output": [
+				"storybook-static"
+			],
+			"dependencies": [
+				"import-wp-css-for-storybook",
+				"dependencyOutputs",
+				"../../plugins/woocommerce-blocks:build:project:storybook"
+			]
```

Comment:
While `wireit` _does_ support script dependencies, thus far, we've only used them to share file input references. When you use `dependencies` like this it has `wireit` manage the task orchestration rather than `pnpm`. This should probably just be part of the script.

Reply:
Thanks, turns out it wasn't necessary anyway as the `pnpm` script already ran build command command of `woocommerce-blocks`. Removed it here: https://github.com/woocommerce/woocommerce/pull/51701/commits/9e04d2312428033bbf35b69dd9396f238c982210